wgpu-types/src/lib.rs

```diff
@@ -855,7 +855,7 @@ impl StencilStateDescriptor {
     pub fn is_enabled(&self) -> bool {
         (self.front != StencilStateFaceDescriptor::IGNORE
             || self.back != StencilStateFaceDescriptor::IGNORE)
-            && self.read_mask != 0
+            && (self.read_mask != 0 || self.write_mask != 0)
```

Comment:
@kvark this is the check I was also wondering about in https://github.com/gfx-rs/wgpu/pull/873#discussion_r469667951

Reply:
yeah, sorry, looks like I screwed this up 😓

Reply:
No problem :) I just wanted to link it here in case it was intentional